src/test/java/com/squareup/javapoet/MethodSpecTest.java

```diff
@@ -416,4 +417,47 @@ static void staticMethod() {
         + "  codeWithNoNewline();\n"
         + "}\n");
   }
+
+  @Test public void controlFlowWithNamedCodeBlocks() {
+    Map<String, Object> m = new HashMap<>();
+    m.put("field", "valueField");
+    m.put("threshold", "5");
+
+    MethodSpec methodSpec = MethodSpec.methodBuilder("method")
+            .beginControlFlow(named("if ($field:N > $threshold:L)", m))
```

Comment:
Please change continuation indents to 4 spaces to be consistent with the rest of the codebase.

Reply:
Otherwise tests look good!

Reply:
Done. For the record, that file mixes continuation indents of 2, 4, and 8 spaces.

Reply:
Thanks! Yeah, should be fixed. 4 spaces is correct.